File: gestor-biblioteca-api/src/main/java/br/hendrew/gestor_biblioteca/enums/Status.java
package br.hendrew.gestor_biblioteca.enums;

import br.hendrew.gestor_biblioteca.interfaces.Description;
import lombok.Getter;

@Getter
public enum Status implements Description {

    ATIVO("Ativo"),
    BAIXADO("Baixado");


    private final String description;

    Status(String description) {
        this.description = description;
    }
}

File: gestor-biblioteca-web/src/app/modules/cadastro/emprestimo/models/enums/status.ts
export enum Status {
    ATIVO = "Ativo",
    BAIXADO = "Baixado",
}

export interface LabelValue {
    label: string;
    value: string;
}

export const StatusLabelValue = [
    { label: 'Ativo', value: 'ATIVO' },
    { label: 'Baixado', value: 'BAIXADO' }
  ];
